Fig.Core/IniFileSettingsSource.cs

```diff
@@ -53,7 +53,7 @@ private static string PrependSection(string section, string key)
         {
             if (!string.IsNullOrEmpty(section))
             {
-                key = section + ":" + key;
+                key = section + "." + key;
```

Comment:
I think there is a _separator field in this class, we should either use it or delete it. I vote for delete it

Reply:
Both `AppSettingsJsonSource.cs` and `EnvironmentVarsSettingsSource` have this. Is that what you are referring to?

I sorta like keeping separate now that I see what you did previously. I'm fine either way, but can put it in this file if you'd like.

Thinking further on this, I wonder if it may be good to have a default separator on the SettingsSource base class and have that something that can be configurable or overridable.

Reply:
> Both `AppSettingsJsonSource.cs` and `EnvironmentVarsSettingsSource` have this. Is that what you are referring to?
Yes, I think so. I didn't have the codebase at hand when writing the comment.

But I think we are good with a dot separator, probably no reason to ever change it. But it does make sense to define it just once and the SettingsSource seems like the reasonable owner of that piece of information.